containers/flatpak/cockpit-client.metainfo.xml.in

```diff
@@ -18,6 +18,21 @@
     </p>
   </description>
 
+  <screenshots>
+    <screenshot>
+      <caption>Cockpit Client login screen</caption>
+      <image>http://cockpit-project.org/images/flatpak/flatpak-screenshot-login.png</image>
```

Comment:
https s'il te plaît -- they are redirected to https anyway, so let's save the user the extra redirection.

Reply:
oops.  thanks!